packages/core/src/App/Components/Elements/LandscapeBlocker/landscape-blocker.tsx

```diff
@@ -1,21 +1,29 @@
 import React from 'react';
 import { useLocation } from 'react-router-dom';
-import { isDisabledLandscapeBlockerRoute, isTabletOs, routes } from '@deriv/shared';
+import { isDisabledLandscapeBlockerRoute, isMobileOs, isTabletOs, routes } from '@deriv/shared';
 import { observer, useStore } from '@deriv/stores';
 import LandscapeBlockerSvg from 'Assets/SvgComponents/settings/landscape-blocker.svg';
 import './landscape-blocker.scss';
+import { useDevice } from '@deriv-com/ui';
 
 const LandscapeBlocker = observer(() => {
     // need to check for wallet account and don't hide landscape blocker for users migrated to wallets
     const {
         client: { has_wallet },
     } = useStore();
+    const { isMobile } = useDevice();
     const location = useLocation();
     const pathname = location?.pathname;
     const is_hidden_landscape_blocker = isDisabledLandscapeBlockerRoute(pathname);
     const shouldShowDtraderTabletView = pathname === routes.trade && isTabletOs;
+    const showBlockerDtraderMobileLandscapeView = !isMobile && isMobileOs() && pathname === routes.trade;
```

Comment:
```suggestion
    const showBlockerDtraderMobileLandscapeView = !isMobile && isMobileOs() && pathname.startsWith(routes.trade);
```
should we also cater for `routes.contract`?

Reply:
yeah i was thinking that but i think the bugs are on trading page so i left it at that